### ISSUE TITLE
[MaterialTimePicker] broken layout if reusing MaterialTimePicker

### ISSUE THREAD

**Issue by ehaberkorn** [material-components/material-components-android#1906]
**Description:** I am creating a MaterialTimePicker Instance and showing it in an DatePicker.addOnPositiveButtonClickListener method. The problem is that the layout is broken if I try to show that TimePicker instance again after closing the first one.
![Screenshot_20201127-150152](https://user-images.githubusercontent.com/64204931/100457149-ce20a780-30c1-11eb-9ed6-efc0cb974dc5.png)

**Expected behavior:** If the MaterialTimePicker is called a second time, the layout should be shown as usual.

**Source code:**
````
    MaterialDatePicker.Builder.datePicker()
        val builder = MaterialDatePicker.Builder.datePicker()
        val pickerDialog = builder.build()
        val timePicker =
            MaterialTimePicker.Builder().setInputMode(MaterialTimePicker.INPUT_MODE_CLOCK)
                .setTimeFormat(TimeFormat.CLOCK_24H).build()

        pickerDialog.addOnPositiveButtonClickListener {
            timePicker.show(childFragmentManager, pickerDialog.tag)
        }
        timePicker.addOnPositiveButtonClickListener {

        }

        view.findViewById<Button>(R.id.button_first).setOnClickListener {
            pickerDialog.show(childFragmentManager, "")
        }
````

**Android API version:** API 30

**Material Library version:** 1.3.0-alpha04

**Device:** Google Pixel 3a

To help us triage faster, please check to make sure you are using the [latest version](https://github.com/material-components/material-components-android/releases) of the library.

We also happily accept [pull requests](https://github.com/material-components/material-components-android/pulls).

**Comment by Aldiwildan77** [comment]
confirmed as a bug, I have got the same problem with this, the time picker is broken at the second time to show

**Comment by AbdElraoufSabri** [comment]
Just happened to me right now

**Comment by sibisasvin** [comment]
is there any improvement ?

**Comment by brinsche-fkt42** [comment]
A duplicate of this bug (https://github.com/material-components/material-components-android/issues/1995) is reportedly fixed in 1.4.0-alpha01 but I haven't had time to test it

**Comment by cyb3rko** [comment]
As #1995 was closed this can also be closed because this question is just a duplicate.
